pypeit/core/wavecal/fitting.py

```diff
@@ -73,7 +163,8 @@ def iterative_fitting(spec, tcent, ifit, IDs, llist, disp,
     # Fit
     n_order = n_first
     flg_quit = False
-    fmin, fmax = -1., 1.
+    #fmin , fmax = -1.0, 1.0
+    fmin, fmax = 0.0, float(nspec-1)
```

Comment:
Hmm..  There is often good reason to keep fmin,fmax in a range like 0,1.
You otherwise may bias the fit to the high pixel values.

Do you have good reason to do otherwise here?

Reply:
The codes scale to fmin, fmax internally when they do the fits, and scale the variables internally when they compute the fits. The only thing I've done is to remove the explicit rescaling here and leave it internal to the codes. That then allows us to store the actual pixels where lines were found in output, rather than re-scaled values. 

The current implementation of robust_polyfit is super error prone, because the user is not explicitly forced to trace the fmin,fmax they used. I think it should return a dict, and operate on a dict that keeps this information and plan to change that in a future PR when I replace robust_polyfit with robust_polyfit_djs.